### PR TITLE
[hi] Localize en/docs/contribute/new-content/open-a-pr.md

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #36465

Note: Please note that the following PR will receive a number of commits before squashing in order to get a better understanding of the changes that are made, rather than reviewing a PR consisting of 1 single commit with 400+ document lines.

It is kindly requested to go through the file changes and suggest the desired changes. The next commit will add the suggestions along with more file content in Hindi.

Thank you!

### CODE REVIEW

**Comment by NitishKumar06** [comment]
I'm closing this PR. Please refer to this PR: https://github.com/kubernetes/website/pull/36637 since the base branch has been updated

File: content/hi/docs/contribute/new-content/open-a-pr.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+{{< note >}}
+**कोड डेवलपर्स:**:यदि आप किसी के लिए एक नई सुविधा का दस्तावेजीकरण कर रहे हैं
+आगामी कुबेरनेट्स रिलीज, देखना
+[दस्तावेज़ एक नई सुविधा](/docs/contribute/new-content/new-features/)|
+{{< /note >}}
+
+नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (पीआर) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) में सभी आवश्यकताओं का पालन करते हैं|
+
+यदि आपका परिवर्तन छोटा है, या आप git से अपरिचित हैं, तो [GitHub का उपयोग करके परिवर्तन](#changes-using-github) पढ़ें किसी पृष्ठ को संपादित करने का तरीका जानने के लिए।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/open-a-pr.md:20]:
```suggestion
यदि आपका परिवर्तन छोटा है, या आप git से अपरिचित हैं, तो किसी पृष्ठ को संपादित करने का तरीका जानने के लिए [GitHub का उपयोग करके परिवर्तन](#changes-using-github) पढ़ें।
```

File: content/hi/docs/contribute/new-content/open-a-pr.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- अवलोकन  -->
+
+{{< note >}}
+**कोड डेवलपर्स:**:यदि आप किसी के लिए एक नई सुविधा का दस्तावेजीकरण कर रहे हैं
+आगामी कुबेरनेट्स रिलीज, देखना
+[दस्तावेज़ एक नई सुविधा](/docs/contribute/new-content/new-features/)|
+{{< /note >}}
+
+नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (पीआर) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) में सभी आवश्यकताओं का पालन करते हैं|
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/open-a-pr.md:18]:
```suggestion
नए सामग्री पृष्ठों में योगदान करने या मौजूदा सामग्री पृष्ठों में सुधार करने के लिए, पुल अनुरोध (PR) खोलें। सुनिश्चित करें कि आप [शुरू करने से पहले](/docs/contribute/new-content/overview/#before-you-begin) अनुभाग में सभी आवश्यकताओं का पालन करते हैं।
```

**Review by amolmote — CHANGES_REQUESTED** [review]
I am done with my review, @NitishKumar06 please look into it what do you think about 
this suggestions, if you think these changes are valid then commit it directly.

**Comment by amolmote** [comment]
@NitishKumar06, you can still update the base branch here then why you are closing this issue.

**Comment by NitishKumar06** [comment]
@amolmote This PR was failing checks due to which I thought of raising a new PR.

File: content/hi/docs/contribute/new-content/open-a-pr.md (hunk 1 above)
Comment (NitishKumar06) [reply content/hi/docs/contribute/new-content/open-a-pr.md:20]: Thank you for the suggestions! I'll add a new commit into the new PR as mentioned above.

File: content/hi/docs/contribute/new-content/open-a-pr.md (hunk 2 above)
Comment (NitishKumar06) [reply content/hi/docs/contribute/new-content/open-a-pr.md:18]: Done! All the suggestions that you suggested were up to the mark! Thank you for reviewing and suggesting it.

**Comment by amolmote** [comment]
@bishal7679  he already opened another pr for same issue. @NitishKumar06  can you please close this pr if you are not working on this pr.

**Pull request closed without merging**